components/Address.jsx

```diff
@@ -32,9 +32,11 @@ const Address = ({ address, networkId }) => {
 Address.propTypes = {
   address: PropTypes.string.isRequired,
   networkId: PropTypes.string,
+  customExplorerUrl: PropTypes.string,
 };
 
 Address.defaultProps = {
   networkId: 'ethereum',
+  customExplorerUrl: undefined,
```

Comment:
no need for this, customExplorerUrl will be undefined if it's not set by someone